Misc/NEWS.d/next/Core and Builtins/2021-01-20-22-31-01.bpo-40176.anjyWw.rst

```diff
@@ -0,0 +1,2 @@
+Syntax errors for unterminated string literals now points to the start,
+instead of the EOF/EOL. Patch by Batuhan Taskaya.
```

Comment:
What about:

Syntax errors for unterminated string literals now point to the begging of the unclosed quote at the start of the string instead of reporting EOF/EOL.